src/liveEditor/listeners/index.ts

```diff
@@ -0,0 +1,72 @@
+import EventListenerHandlerParams from "./params";
+
+import handleMouseClick from "./mouseClick";
+import handleMouseOver from "./mouseOver";
+import handleMouseHover from "./mouseHover";
+import VisualEditorGlobalState from "../globals";
+
+interface addEventListenersParams
+    extends Omit<EventListenerHandlerParams, "event" | "eventDetails"> {}
+
+interface RemoveEventListenersParams
+    extends Omit<EventListenerHandlerParams, "event" | "eventDetails"> {}
+
```

Comment:
If we are going to keep these empty, let's convert them to a type. Please check eslint errors.